### PR TITLE
Added svg upload command

### PR DESCRIPTION
**Pull request by sheralim012**
- Added a management command that can bulk upload SVG
- Only works if a certain folder structure is followed
```
  - svg-dir
    - dir1
      - abc.svg
    - dir2
      - xyz.svg
```
- usage: `python manage.py load_svg --svg-dir <path-to-svg-dir>`

### CODE REVIEW

File: iogt_content_migration/management/commands/load_svg.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--svg-dir',
+            required=True,
+            help='Path to IoGT SVG directory'
+        )
+
+    def handle(self, *args, **options):
+        self.svg_dir = options.get('svg_dir')
+        self._clear()
```
Comment (ehmadzubair) [inline iogt_content_migration/management/commands/load_svg.py:19]: better to have this behind a flag

File: iogt_content_migration/management/commands/load_svg.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        self._load_svg()
+
+    def _clear(self):
+        Svg.objects.all().delete()
+
+    def _open_svg_file(self, svg_path):
+        try:
+            return open(svg_path, 'rb')
+        except:
+            self.stdout.write(f'SVG file not found: {svg_path}.')
```
Comment (ehmadzubair) [inline iogt_content_migration/management/commands/load_svg.py:29]: style.WARN or style.ERROR

File: iogt_content_migration/management/commands/load_svg.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    def _load_svg(self):
+        for dir in os.listdir(self.svg_dir):
+            dir_path = os.path.join(self.svg_dir, dir)
+            if os.path.isdir(dir_path):
+                for svg_file in os.listdir(dir_path):
+                    svg_path = os.path.join(dir_path, svg_file)
+                    file = self._open_svg_file(svg_path)
+                    if file:
+                        svg = Svg.objects.create(title=svg_file.strip('.svg'), file=File(file, name=svg_file))
+                        svg.tags.add(dir)
```
Comment (ehmadzubair) [inline iogt_content_migration/management/commands/load_svg.py:40]: Add a line for successful load

File: iogt_content_migration/management/commands/load_svg.py (hunk 2 above)
Comment (sheralim012) [reply iogt_content_migration/management/commands/load_svg.py:29]: done

File: iogt_content_migration/management/commands/load_svg.py (hunk 3 above)
Comment (sheralim012) [reply iogt_content_migration/management/commands/load_svg.py:40]: done

File: iogt_content_migration/management/commands/load_svg.py (hunk 1 above)
Comment (sheralim012) [reply iogt_content_migration/management/commands/load_svg.py:19]: done

**Comment by sheralim012** [comment]
@ehmadzubair changes addressed.

**Pull request merged**